src/wallets/hardware/ledger/appPaths.js

```diff
@@ -148,8 +148,8 @@ const appList = [
   },
   {
     network: TOMO,
-    prefixes: ["m/44'/889'"],
-    paths: [tomoChain]
+    prefixes: ["m/44'/889'", "m/44'/60'"],
+    paths: [tomoChain, ledgerEthereum, ledgerLiveEthereum, ethereum]
```

Comment:
These paths wont work with Tomochain ledger app! :(
If you want to use these paths user has to open the ETH app then use these paths

Reply:
Yep, TOMO holder use both ETH app, and TomoChain App. So MEW should support the both so that they can unlock the wallet

Reply:
We'll wait for your PR on Ledger's end to get merged and release to merge this PR.

Reply:
There are many TOMO Holders using Ledger Nano S with hd path 44/60, so really hope you guys can help to process this PR, It is very helpful for them to unlock their wallet.